test/unit/input_convertors.py

```diff
@@ -98,4 +98,4 @@ def _map_into_single_column(matrix):
     if len(np.array(matrix).shape) == 1:
         return np.array(matrix)
 
-    return _compress_multiple_sensitive_features_into_single_column(matrix)
+    return _compress_multiple_sensitive_features_into_single_column(np.array(matrix))
```

Comment:
@romanlutz ?

Reply:
This was something that I am proposing(as of now) to fix the tests with these changes, specifically, I wanted some feedback on this because `drop` is not present in `numpy`, as a result, we would need to ensure the input `X` is a `pandas` object.